src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health, name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  removeDead() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(health, name);
+  }
+
+  bite(individual) {
+    if (individual instanceof Herbivore && !individual.hidden) {
+      individual.health -= 50;
+
+      if (individual.health <= 0) {
+        individual.removeDead();
```

Comment:
[STATIC METHOD CALL] - The `removeDead` method is being called on an instance of the `Animal` class. Since `removeDead` should be a static method, it should be called using the class name, like `Animal.removeDead()`.